x/escrow/msg_test.go

```diff
@@ -34,7 +34,7 @@ func TestCreateEscrowMsg(t *testing.T) {
 	// invalid
 	d := weave.Condition("foobar")
 
-	timeout := time.Now()
+	timeout := weave.AsUnixTime(time.Now())
```

Comment:
A native `Now()` on the new type would be nice

Reply:
I am not sure what you are suggesting. There are no constructors in Go. I could add `weave.UnixTimeNow()` function, but I think having smaller API is in this case better.

Reply:
Ah, so only instance methods are possible (like .IsZero(), .Validate()) in UnixTime but no factory functions, right? I see, yeah, not so important. Smaller API is good.